web/pdf_history.js

```diff
@@ -191,30 +192,31 @@
 
     _pushOrReplaceState: function pdfHistory_pushOrReplaceState(stateObj,
                                                                 replace) {
-//#if CHROME
-      // history.state.chromecomState is managed by chromecom.js.
-      if (window.history.state && 'chromecomState' in window.history.state) {
-        stateObj = stateObj || {};
-        stateObj.chromecomState = window.history.state.chromecomState;
+      if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('CHROME')) {
```

Comment:
Nit: shouldn't this condition be `typeof PDFJSDev !== 'undefined' && PDFJSDev.test('CHROME')`, since it only matters in `CHROME` builds?